src/main/g8/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'application'
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile 'com.typesafe.akka:akka-http_2.12:$akka_http_version$'
+    compile 'com.typesafe.akka:akka-stream_2.12:$akka_version$'
+    testCompile 'com.typesafe.akka:akka-http-testkit_2.12:$akka_http_version$'
+    testCompile 'org.scalatest:scalatest_2.12:3.0.1'
```

Comment:
Hmm, yeah I wonder - would be good to not have to not have java developers have to include scalatest explicitly; Ticket please I think